### PR TITLE
dysk: add page

### PR DESCRIPTION
**Pull request by lloyd-dada**
Added new page for dysk in linux

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
- dysk 2.8.2

### CODE REVIEW

File: pages/linux/dysk.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# dysk
+
+> Display filesystem information in a pretty table.
+> More information: <https://dystroy.org/dysk>.
+
+- Standard overview of your usual disks:
```
Comment (spageektti) [inline pages/linux/dysk.md:6]:
```suggestion
- Get a standard overview of your usual disks:
```

File: pages/linux/dysk.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`dysk`
+
+- Sort by free size:
+
+`dysk --sort free`
+
+- Filter to include only HDD disks:
```
Comment (spageektti) [inline pages/linux/dysk.md:14]:
```suggestion
- Include only HDD disks:
```

File: pages/linux/dysk.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`dysk --filter 'disk = HDD'`
+
+- Filter to exclude SSD disks:
```
Comment (spageektti) [inline pages/linux/dysk.md:18]:
```suggestion
- Exclude SSD disks:
```

File: pages/linux/dysk.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# dysk
+
+> Display filesystem information in a pretty table.
```
Comment (spageektti) [inline pages/linux/dysk.md:3]:
```suggestion
> Display filesystem information in a table.
```

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution. I left some suggestions based on our style guide.

File: pages/linux/dysk.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Include only HDD disks:
+
+`dysk --filter 'disk = HDD'`
+
+- Exclude SSD disks:
+
+`dysk --filter 'disk <> SSD'`
+
+- Filter for high utilization or low free space:
```
Comment (spageektti) [inline pages/linux/dysk.md:22]:
```suggestion
- Display disks with high utilization or low free space:
```

**Review by spageektti — APPROVED** [review]
LGTM, thanks.

**Comment by lloyd-dada** [comment]
> LGTM, thanks.

Thanks for fixing! Sorry for my poor interpretation of the style guide.

**Pull request merged**